app/models/film.rb

```diff
@@ -8,4 +8,7 @@ class Film < ApplicationRecord
     join_table: 'films_people',
     foreign_key: 'film_id',
     association_foreign_key: 'person_id'
+
+  has_many :ratings
+  has_many :users, through: :ratings
```

Comment:
Ah! Yes!

It'll allow us to easily add `field :critics` in GraphQL type if we name the association correctly.